app/src/main/java/com/example/splendormobilegame/activities/GameActivity/GameActivity.java

```diff
@@ -312,13 +316,59 @@ public void onClick(DialogInterface dialog, int which) {
                 })
                 .show();
     }
-    public void setupCardsFirstTierRecyclerView(){
+
+    public void showDeckReserveDialog(UUID cardUuid) {
```

Comment:
This still expects UUID as a parameter, but should expect int instead.

Reply:
yup, I missed that